cmd/initializer/dataset/Dockerfile

```
@@ -10,7 +10,7 @@ COPY pkg/initializer pkg/initializer
 RUN pip install -r requirements.txt
 
 # Git is needed for the Kubeflow SDK to download JobSet Python models.
-RUN apk update && apk add --no-cache git
+RUN apt update && apt install -y git
```

```suggestion
RUN apt-get update && apt-get install -y git
RUN apt-get clean && rm -rf /var/lib/apt/lists/*
```

We want to clean up source lists since we want to make images as minimal as possible.
Additionally, `apt` is an interactive command; in script, `apt-get` is more stable and appropriate.